### ISSUE TITLE
Completely useless, harmful for sites and anti-visitor

### ISSUE THREAD

**Issue by Sommerwiesel** [google/recaptcha#502]
Hello, longterm webdeveloper and nowadays Linux Cloud admin here,
in all my years, google recaptcha has been and (taking into account how googles ability to provide good services has stagnated in the past years) probably will always be the worst thing to protect your site.
It is not only completely inefficient in preventing sofisticated bots from wrecking havok, it also blocks out roughly 1/3 of legit user interaction.
I have carefully analyzed statistics (before and while using recaptcha) and have found out, that when a legitimate user is prompted with this recaptcha, theres a 1/3 chance he'll just get annoyed and leave.
And I've been there myself.

After just having to sit through 5(!!!) recaptcha images in an online store, I've given up ordering. They have lost around 200,- € of revenue I will spend on a shop that has the same item but without the god damn google recaptcha.

During my past career as a webdeveloper, everytime we switched from google recaptcha to a more simple version (and instead used proper filtering of malicious traffic), our site interactions (and I'm talking about legitimate users here, not bots) have gone back up.

TLDR - google recaptcha is both harmful to site owners and visitors while also completely failing to provide what it has been created to do: stop bots.

Please please burry this service already. We're all sick and tired of it.

**Comment by AlexRazzoR** [comment]
Do you know about a solution like CleanTalk Anti-Spam?
https://github.com/CleanTalk/php-antispam

**Comment by Sommerwiesel** [comment]
> Do you know about a solution like CleanTalk Anti-Spam? https://github.com/CleanTalk/php-antispam

My post is not about finding an alternative. My post is about google recaptcha bring shit. Sadly, it's used in 2/3 of all Websites. Which is horrible for visitors

**Comment by tommygooden** [comment]
I agree with you as a web developer myself. However, as a software engineer, I think we should always seek solutions. I really think Google should take into consideration what you're feeling and understand it is a feeling shared right around. However, I don't think throwing away all the progress and knowledge that was gained by this project is the right way. Would you be interested in building something different? And how would you go about it?

**Comment by tommygooden** [comment]
I agree with you as a web developer myself. However, as a software engineer, I think we should always seek solutions. I really think Google should take into consideration what you're feeling and understand it is a feeling shared right around. However, I don't think throwing away all the progress and knowledge that was gained by this project is the right way. Would you be interested in building something different?

**Comment by rowan-m** [comment]
Repo is just for PHP client issues. This is about the reCAPTCHA service.
